Project page header

anishTP opened the pull request:
- Moved the register block to the project-banner_right
- Added a venue card to the right_col below navbar on the project page.
- Changed order of the navbar render on mobile

jace commented:
The layout needs cleanup for some cases:

Member box alignment and missing icon for venue:
<img width="1185" alt="image" src="https://github.com/hasgeek/funnel/assets/26695/82e2969f-9e6d-40b5-8aad-bf3b0677db52">

Everything is left aligned when there's no image, but the button's visual alignment is off:
<img width="1187" alt="image" src="https://github.com/hasgeek/funnel/assets/26695/8ad92b1f-65dc-46ec-94f3-71da55f40d8d">

Here the colour contrast works in the button's favour, but it needs margin from the text above it. The second button declaring "You are a member" is confusing as it has no relevance to this header:
<img width="1176" alt="image" src="https://github.com/hasgeek/funnel/assets/26695/6543213d-6947-400e-a50c-c830c40f2d03">

anishTP commented:
> The layout needs cleanup for some cases:
> 
> Member box alignment and missing icon for venue: <img alt="image" width="1185" src="https://private-user-images.githubusercontent.com/26695/326179751-82e2969f-9e6d-40b5-8aad-bf3b0677db52.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzODAzOTksIm5iZiI6MTcxNDM4MDA5OSwicGF0aCI6Ii8yNjY5NS8zMjYxNzk3NTEtODJlMjk2OWYtOWU2ZC00MGI1LThhYWQtYmYzYjA2NzdkYjUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDA4NDEzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmN2UwY2Y5YzhkMTZlNDQyYjNlYTk1MDRkOTA3YzM2Y2U4YzUyOGIxMmI5ZWY0Mzg0NTFjZmU3OGM3ODZiOWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.K6Yhi8n1c36EsgorlVjUHyI1Eh1Dsud0A82homUCkZI">

Changed the size of the button to full-width when its the only child.
Replaced map-pin with map-marker-alt icon for the venue.
<img width="1053" alt="Screenshot 2024-04-29 at 5 11 52 PM" src="https://github.com/hasgeek/funnel/assets/119032387/0b152891-fa7e-4404-9024-0605cb40130b">


> 
> Everything is left aligned when there's no image, but the button's visual alignment is off: <img alt="image" width="1187" src="https://private-user-images.githubusercontent.com/26695/326179859-8ad92b1f-65dc-46ec-94f3-71da55f40d8d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzODAzOTksIm5iZiI6MTcxNDM4MDA5OSwicGF0aCI6Ii8yNjY5NS8zMjYxNzk4NTktOGFkOTJiMWYtNjVkYy00NmVjLTk0ZjMtNzFkYTU1ZjQwZDhkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDA4NDEzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk2MDc4MWJlMTdhYzVkYThhM2MzMGNmNWJiMGVkNzllMDI3ZWM4ODZhM2Q0M2ZjYTY3N2U1NzRjZTE3YjUwYzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.t3W0BF4n6ZQBcK-R1EcpYlYLh1ehxJpTzM3bX9xC8Oo">

Added padding to above the button when there is no image and start-date. Also made it full-width when its the only child.

<img width="1064" alt="Screenshot 2024-04-29 at 5 10 52 PM" src="https://github.com/hasgeek/funnel/assets/119032387/2883a420-4d60-4e5b-bd61-e8ed76361255">

> 
> Here the colour contrast works in the button's favour, but it needs margin from the text above it. The second button declaring "You are a member" is confusing as it has no relevance to this header: <img alt="image" width="1176" src="https://private-user-images.githubusercontent.com/26695/326179957-6543213d-6947-400e-a50c-c830c40f2d03.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQzODAzOTksIm5iZiI6MTcxNDM4MDA5OSwicGF0aCI6Ii8yNjY5NS8zMjYxNzk5NTctNjU0MzIxM2QtNjk0Ny00MDBlLWE1MGMtYzgzMGM0MGYyZDAzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI5VDA4NDEzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4YTgxNWRiZjA5ZGVkOWMwMjRhMjhkMWUwYjliYjc2MjQ5Njc2MjUxMDJhYmUxNGY3NjhlNWI3OWE5ZDYwZWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LtbETWEFxM6r8Z-oH10AxKRRcVYvpRuoK5WRYAVV1N8">

The membership button could be removed since the header already has a label showing membership status. But it is necessary when the user is not a member.
<img width="1065" alt="Screenshot 2024-04-29 at 5 10 37 PM" src="https://github.com/hasgeek/funnel/assets/119032387/58a9cf1c-5992-4ea6-be4f-601a0e2861e9">

jace commented on funnel/templates/project_layout.html.jinja2:
```diff
@@ -358,7 +338,7 @@
           {%- elif sponsorship.member.description %}
             <div class="mui--text-body2 mui--text-light margin-bottom">
-              {{ sponsorship.member.description|preview(min=200, max=300) }}
+              {{ sponsorship.member.description.html|preview(min=200, max=300) }}
```
This has been fixed in Baseframe. Using `.html` for the `|preview` filter is no longer necessary.
